docs/DafnyRef/examples/Example-TwoState.dfy

```diff
@@ -0,0 +1,18 @@
+
+// This file only exists to include these examples as a test case.
```

Comment:
Do you know why is this here as a test-case but not used in the reference manual?

Reply:
Yup, it's just because the section that uses these examples (https://dafny-lang.github.io/dafny/DafnyRef/DafnyRef#sec-two-state) doesn't ever actually define the `Cell` class, but it has to be defined here in order for the examples to resolve as complete Dafny programs.